### DISCUSSION TITLE
Changing NICs in a distributed deployment without rebuilding

### DISCUSSION THREAD

**Discussion by cyb3rslack3r** [Security-Onion-Solutions/securityonion#3981] [Unsupported Versions]
I have a distributed deployment running 2.3.1 with 2 Forward Sensors, 1 Manager and 3 Search Nodes.  I initially configured the 2 Forward Sensors to use one 10gig port for managing and the other for pcap collection.  I have a scenario where I might need to reconfigure both 10gig ports (only 2 10gig ports on server) for collection and then use a 1 gig Ethernet port for managing.  I don't want to rebuild both these sensors to do this (or the whole deployment), but want to make sure it doesn't screw up communication to the SO Manager if I change configuration files or re-run so-setup.  What would be a best practice way to accomplish this?

Bonus question: Is there a way in SO configs to concatenate the 2 pcap ingest ports to stitch the traffic from a TAP together to maintain session data when querying the PCAP? Trying to avoid using a TAP aggregator for this

Thanks

**Comment by TOoSmOotH** [comment]
The first thing I would do is update to the latest code.  

You can change the interface as long as you keep the same IP. You will need to modify the minion file on the manager as well as the corresponding minion file in the data pillar. 

Once that is done you add your new monitor interface to the bond. The bond takes care of stitching the traffic together.
